Automattic/wp-calypso
# Email Subscriptions: Create DeliveryWindowInput component

Resolves #74470

## Proposed Changes

* Creates DeliveryWindowInput component

## Testing Instructions

* Deploy this PR on your local env
* Open http://calypso.localhost:3000/subscriptions/settings
* Verify if the page is rendered with the Delivery Window field

<img width="644" alt="Screenshot 2023-03-17 at 16 17 54" src="https://user-images.githubusercontent.com/3113712/226003485-5e4011ad-8b95-499b-bdea-430c3e762484.png">

## Pre-merge Checklist
